src/app/index.js

```diff
@@ -1,2 +1,13 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
 require('./scss/styles.scss');
-console.log("Meaningful Message!!!!"); 
\ No newline at end of file
+
+class ReactHeader extends React.Component{
+  render(){
+    return (
+      <div>
+        <h1>Hello World!</h1>
+      </div>);
+  }
+}
+ReactDOM.render(<ReactHeader/>, document.querySelector(".center"));
```

Comment:
add a space before `/>` also replace the root element with `<div id="root"></div>` in the HTML also